netbsd/Platform.c

```diff
@@ -200,7 +200,7 @@ int Platform_getMaxPid() {
 
 double Platform_setCPUValues(Meter* this, int cpu) {
    const NetBSDProcessList* npl = (const NetBSDProcessList*) this->pl;
-   const CPUData* cpuData = &npl->cpus[cpu];
+   const CPUData* cpuData = &(npl->cpuData[cpu]);
```

Comment:
```suggestion
   const CPUData* cpuData = &npl->cpuData[cpu];
```

Those parens are not needed …

Reply:
Should be fixed in the latest commit.